pkg/acceptance/compose_test.go

```diff
@@ -34,6 +35,7 @@ func TestComposeGSSPython(t *testing.T) {
 }
 
 func TestComposeFlyway(t *testing.T) {
+	skip.UnderArm64(t)
```

Comment:
would be nice to have a comment or description here for why it's skipped.

Reply:
Agree, actually I initially skipped it because `flyway:flyway6` image didn't support `linux/arm64`. Now I see that `linux/arm64` is supported since `8.4.3` so I will update the image and unskip the test instead